forge-gui/res/cardsfolder/upcoming/sadistic_shell_game.txt

```diff
@@ -0,0 +1,8 @@
+Name:Sadistic Shell Game
+ManaCost:4 B
+Types:Sorcery
+A:SP$ RepeatEach | RepeatPlayers$ Opponent | RepeatSubAbility$ DBChooseCard | SubAbility$ DBDestroy | InTurnOrder$ True | StackDescription$ SpellDescription | SpellDescription$ Starting with the next opponent in turn order, each player chooses a creature you don’t control. Destroy the chosen creatures.
```

Comment:
that is actually incorrect since the choices should still be made by all players

looks like this card will be a bit complicated to get right, let me know if you'd rather I take over at any point

Reply:
I'm not sure what you mean. Each opponent makes a choice and then the cards are destroyed. If you could clarify I can look into it a bit more.

Reply:
it's not identical to _Manifold Insights_:
"each **player** chooses"
only who starts is defined differently than the rules default

Reply:
Oh man. I even played this card at my table the other night and we definitely did not let me choose. That is just a misread on mine and my pods part. I'll fix this right away.